src/games/strategy/triplea/printgenerator/CountryChart.java

```diff
@@ -89,11 +89,8 @@ protected void saveToFile(final PlayerID player, final PrintGenerationData print
         countryFileWriter.write("\r\n");
       }
       countryFileWriter.close();
-    } catch (final FileNotFoundException e) {
-      e.printStackTrace();
     } catch (final IOException e) {
-      // TODO Auto-generated catch block
-      e.printStackTrace();
+      ClientLogger.logError("Failed Saving to File " + outFile.toString(), e);
```

Comment:
`.getAbsolutePath()` is what I would have chosen. I think `toString()` likely will work too.